brand/models/commentary.py

```diff
@@ -7,6 +7,7 @@
 from django_countries.fields import CountryField
```

Comment:
It looks like CountryField and Enum are unused. I has my git blame on it, but don't hesitant to remove stuff like that as long as the tests still pass. You can check to make sure no one is importing this in a roundabout way by searching `from Commentary import CountryField`. If there aren't matches then you're probably good to remove it